plugins/chat/assets/javascripts/discourse/components/chat/modal/edit-channel-name.js

```diff
@@ -48,9 +48,9 @@ export default class ChatModalEditChannelName extends Component {
   }
 
   @action
-  onChangeChatChannelName(title) {
+  onChangeChatChannelName(event) {
     this.flash = null;
-    this.#debouncedGenerateSlug(title);
+    this.#debouncedGenerateSlug(event?.target?.value);
```

Comment:
I did the most conservative `?.` here but that is probably not actually necessary, the function is not called anywhere else so it should always have `event.target.value`. Alternatively we could probably just use `this.editedName` here, but I wasn't sure if there are going to be subtle timing differences so I left it, but someone can verify that and do the refactor if desired.

Reply:
Yes fine with this for now thanks